### DISCUSSION TITLE
@types/gulp-eslint-if-fixed

### DISCUSSION THREAD

**Discussion by Dulux-Oz** [DefinitelyTyped/DefinitelyTyped#58697] [Request a new @types package] [answered]
Hi All,

Could someone help a (TypeScript) noob (again :-) ) with some advice, please, by answering one (or more) of the following questions:

1. Does anyone have the TypeScript def file for gulp-eslint-if-fixed, and may I please have a copy?
2. Could someone please write the TypeScript def file for gulp-eslint-if-fixed?
3. Could someone help me to write the TypeScript def file for gulp-eslint-if-fixed?
4. Could someone point me towards how to use gulp-eslint-if-fixed "as it is" with TypeScript?

(FTR: I looked on NPM but couldn't find anything "newer" or "more appropriate".)

Thanks in advance,

Dulux-Oz

**Comment by fasttime** [comment, marked as answer]
If you are using the current version of gulp-eslint-new, you don't need gulp-eslint-if-fixed. Fix your code by piping to `gulpESLintNew.fix()`. See https://github.com/fasttime/gulp-eslint-new#gulpeslintnewfix for usage.

**Reply by Dulux-Oz** [reply]
Cool, thanks for that - I missed that when I was updating from `gulp-eslint` to `gulp-eslint-new`.
